codegen.yml

```diff
@@ -26,6 +26,7 @@ generates:
         DataCubeMetadata: "../domain/data#DataCubeMetadata"
         DataCubeObservations: "../domain/data#DataCubeObservations"
         DataCubePreview: "../domain/data#DataCubePreview"
+        Termset: "../domain/data#Termset"
```

Comment:
I think you also need to add this to `app/graphql/resolver-types.ts` down below? Not sure if the current behavior would cause some problems, but it looks like the scalars are defined in both places, I remember having some `any` types when adding them only to one file.

Reply:
Thanks !